docs/references/express/migrate-from-node-to-express.mdx

```diff
@@ -0,0 +1,102 @@
+---
+description: Learn how to migrate from the Clerk's Node SDK to the Express SDK.
+---
+
+# Migrate from Clerk's Node SDK to the Express SDK
+
+## Install the Express SDK
+
+To migrate from the Clerk Node SDK to the Clerk Express SDK, uninstall the Node SDK by running the following command:
+
+<CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+  ```sh filename="terminal"
+  npm uninstall @clerk/clerk-sdk-node
+  ```
+
+  ```sh filename="terminal"
+  yarn remove @clerk/clerk-sdk-node
+  ```
+
+  ```sh filename="terminal"
+  pnpm remove @clerk/clerk-sdk-node
+  ```
+</CodeBlockTabs>
+
+Then, install the Express SDK by running the following command:
+
+<CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+  ```sh filename="terminal"
+  npm install @clerk/express
+  ```
+
+  ```sh filename="terminal"
+  yarn add @clerk/express
+  ```
+
+  ```sh filename="terminal"
+  pnpm add @clerk/express
+  ```
+</CodeBlockTabs>
+
+## Migrate to `requireAuth`
+
+To enforce strict authentication, replace your existing [Node SDK middleware](/docs/backend-requests/handling/nodejs) with the Express SDK middleware, [`requireAuth`](/docs/references/express/require-auth).
+
+`requireAuth` is a middleware function that you can use to protect routes in your Express.js application. This function checks if the user is authenticated or authorized, and returns a 401 status code if they are not.
+
+`clerkMiddleware()` is required to be set in the middleware chain before this util is used, as shown in the following example.
+
+```diff
+- import {
+-  ClerkExpressRequireAuth,
+-  RequireAuthProp,
+-  StrictAuthProp
+- } from '@clerk/clerk-sdk-node';
++ import { clerkMiddleware, requireAuth } from '@clerk/express';
+import express from 'express';
+
+const app = express();
+
+const port = 3000;
+
++ // Apply centralized middleware
++ app.use(clerkMiddleware());
+
+// Define a protected route
+- app.get('/protected', ClerkExpressRequireAuth(), (req, res) => {
+-   res.send('This is a protected route');
+- });
++ app.get('/protected', requireAuth, (req, res) => {
++   res.send('This is a protected route');
++ });
+
+// Start the server and listen on the specified port
+app.listen(port, () => {
+  console.log(`Server is running on http://localhost:${port}`);
+});
+```
+
+### Get authentication state
+
+**Using Node SDK**
```

Comment:
This headline is odd to me - if we're going to have a "using the node sdk" example, we should also have a "using the express sdk" headline - or we can cut them both and word it so that it is clear